src/components/selection_input/selection_input.xml

```diff
@@ -47,7 +48,62 @@
           <t t-call="o-spreadsheet-Icon.TRASH_FILLED"/>
         </button>
       </div>
-
+      <t t-if="hasDisabledRanges">
```

Comment:
Do we really need to copy/paste the whole xml here ? Can't we just apply a style to the disabled ranges ?